docker/files/run/run.sh

```diff
@@ -22,6 +22,9 @@ function apply_configuration() {
   if [ -f "/etc/algod.admin.token" ]; then
     cp /etc/algod.admin.token algod.admin.token
   fi
+  if [ -f "/etc/logging.config" ]; then
```

Comment:
Add this to the "special files" section of the readme? What is the precedence if "TELEMETRY_NAME" is also set?